web/src/features/moreCast2/components/SelectableButton.tsx

```diff
@@ -1,25 +1,58 @@
 import React from 'react'
 
 import { Button } from '@mui/material'
-import { theme } from 'app/theme'
+import { MORECAST_WEATHER_PARAM_COLORS, MoreCastParamColors, theme } from 'app/theme'
+import styled from '@emotion/styled'
+
+const StyledButton = styled(Button)(() => ({
+  marginLeft: theme.spacing(1),
+  '&:hover': {
```

Comment:
Not code related but I made a comment on Figma that Raj hasn't replied to yet. Maybe it's minor but curious what you think.
When hover is white, it's hard to tell when you've actually clicked on the tab. You have to move your cursor off of the button to see if it's changed colour to active/inactive.
Take it or leave it, just a thought

Reply:
Chatted about it and got the green light